src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -120,47 +127,79 @@ class FacetSidebar extends React.Component {
           .reduce((x, y) => x + y, 0)
           .value();
 
+        if (facet.field === 'date') {
+          return (
```

Comment:
What gets returned here can be a component itself. I think for another iteration where we make it functional, that should be a goal. For now this is good.